src/modules/others/timer.h

```diff
@@ -6,15 +6,31 @@
  * @date 2024-12-07
  */
 
+#ifndef __TIMER_H__
+#define __TIMER_H__
+
+#include <globals.h>
 class Timer {
 private:
 
+    int fontSize = 4;
     int duration = 0;
+    int timerX = tftWidth / 2;
```

Comment:
I think constants should be all uppercase so it doesn't get confused with normal variables. 
As an example you can check the tft colors like TFT_RED, TFT_BLACK and other constants from many other libraries

Reply:
It's not a constant however. It's actually changing based off the tft width and height function which can change when you change rotation. This is future proofing to whenever we choose to add portrait.

Reply:
This was actually the solution I used to fix the issue with the fact that we had the TFT based height and width definition as it should have been but then you had width and height as their own definitions which was just inverse of the TFT based definitions. Honestly this is pretty silly and not very dynamic. Also with things like those colors I intend to throw those into enums. Abusing macro definitions like we do is not really the way to go about this